pkg/commands/entrypoints/entrypoints.go

```diff
@@ -153,6 +160,10 @@ func (c *EntrypointsCommand) Flags() *cli.FlagSet {
 			c.parsedFlagMaxDepth = &c.flagMaxDepth
 		}
 
+		if c.flagDir == "" {
+			merr = errors.Join(merr, fmt.Errorf("missing flag: dir is required"))
```

Comment:
```suggestion
			merr = errors.Join(merr, fmt.Errorf("missing flag: -dir is required"))
```

But see also: should this default to CWD?